Magic™Editor.user.js

```diff
@@ -3494,21 +3495,20 @@
         // This is the styling for the diff output.
         $('body').append('<style>' +
                          '.difftitle {' +
-                         '    color: rgb(34, 34, 34);' +
+                         '    color: var(--black);' +
                          '    font-size: 24px;' +
                          '    font-weight: normal;' +
                          '    line-height: 36px;' +
                          '    margin-bottom: 12px;' +
                          '}' +
                          '.diffbody {' +
                          '    white-space: pre-wrap;' +
-                         '    font-family: "courier new", "lucida sans typewriter", mono, monospace' +
                          '}' +
                          '.add {' +
-                         '    background: #CFC;' +
+                         '    background: var(--green-100);' +
```

Comment:
`--green-100` is equal to `#3d6048` in dark mode or `#cae8d4` in light, not `#CCFFCC`.

Reply:
`--green-100` is consistent with the background colour used for diffs in revisions. Matched in the CSS rule for `span.diff-add`